tests/drivers/smbus/smbus_emul/prj.conf

```diff
@@ -1,3 +1,5 @@
 CONFIG_ZTEST=y
 CONFIG_LOG=y
 CONFIG_ENTROPY_GENERATOR=y
+CONFIG_SMBUS=y
```

Comment:
Why do you need this? Is this to set LOG LEVEL?

Reply:
I need it because otherwise I get a linker error for the stuff which I moved into smbus_utils.c And I'm not a big fan of including c-files, therefore I enabled CONFIG_SMBUS.

Reply:
OK, understood. This can also be moved to header.